.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    extends: ['@prezly', '@prezly/eslint-config/react', '@prezly/eslint-config/nextjs'],
+    extends: ['@prezly/eslint-config/nextjs'],
```

Comment:
I updated the configs in v5 to extend each other, so now only a single config for the specific environment is needed.